### PR TITLE
Colombia marriage same sex

### PR DESCRIPTION
**Pull request by mgrassotti**
Supersedes #3059

## Description

Same-sex marriages now allowed under local law.

### Expected changes

- Content update for the 9 same sex outcomes to reflect the fact that same sex marriage is now available under local law

- Content update for the 6 outcomes where the user is in Colombia to update information about contacting local authorities

This doesn't affect the logic

We need to replace the current content for same sex marriage tool outcomes in Colombia with the existing content for opposite sex marriages.

### Before
colombia-before

### After
colombia-after

### Affected paths
1. Where the British national is resident in the UK

  Change:
  /marriage-abroad/y/colombia/uk/partner_local/same_sex 

  To be the same as:
  /marriage-abroad/y/colombia/uk/partner_local/opposite_sex
 
2. Where the British national is resident in Colombia

  Change:
  /marriage-abroad/y/colombia/ceremony_country/partner_local/same_sex 

  To be the same as:
  /marriage-abroad/y/colombia/ceremony_country/partner_local/opposite_sex

3. Where the British national is resident Elsewhere

  Change:
  /marriage-abroad/y/colombia/third_country/partner_local/same_sex 

  To be the same as:
  /marriage-abroad/y/colombia/third_country/partner_local/opposite_sex

### Factcheck

https://smart-answers-pr-3069.herokuapp.com/marriage-abroad

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb [hunk 1]
```diff
@@ -13,15 +13,13 @@ Make an appointment at the British embassy in Bogota to swear an affidavit (writ
 
 ## Fees
 
-<%= render partial: 'consular_fees_table_items.govspeak.erb',
-           collection: calculator.services,
-           as: :service,
-           locals: { calculator: calculator } %>
-
+Service | Fee
```
Comment (chao-xian) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: Partial for fees needs reinstating.

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb [hunk 2]
```diff
@@ -13,15 +13,13 @@ Make an appointment at the British embassy in Bogota to swear an affidavit (writ
...
+- |-
+Affidavit or affirmation for marriage | £50
+ 
 You normally have to pay fees for consular services in the local currency - these are shown in the list of [consular fees for Colombia](/government/publications/colombia-consular-fees).
-
-<%=  render partial: 'how_to_pay.govspeak.erb',
-            locals: {calculator: calculator}%>
+ 
+You can pay by cash or credit card, but not by personal cheque.
```
Comment (chao-xian) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: Partial for how to pay needs reinstating.

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb (hunk 1 above)
Comment (mgrassotti) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: Ok

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb (hunk 2 above)
Comment (mgrassotti) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: Ok

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb (hunk 1 above)
Comment (chao-xian) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: Sorry @mgrassotti - I only commented on the first erb template but the same needs to happen for the others for the Colombia. Sorry I didn't make that clear before. Do you want me to add a comment in the others as well?

Also I think you can remove the:

```
Service | Fee
- |-
Affidavit or affirmation for marriage | £50
```

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb (hunk 2 above)
Comment (chao-xian) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: You can remove the `You can pay by cash or credit card, but not by personal cheque.` line as the partial will render that.

Comment (mgrassotti) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: 👍

File: lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb (hunk 1 above)
Comment (mgrassotti) [inline lib/smart_answer_flows/marriage-abroad/outcomes/countries/colombia/ceremony_country/partner_british/_opposite_sex.govspeak.erb]: 👍

**Comment by mgrassotti** [comment]
@chao-xian ready for review again thanks

**Comment by mgrassotti** [comment]
@chao-xian sure no problem, it should be done

**Comment by mgrassotti** [comment]
@chao-xian 👍 done

**Pull request merged**